src/main/java/mate/academy/service/impl/EmailValidatorServiceImpl.java

```diff
@@ -1,2 +1,19 @@
-package mate.academy.service.impl;public class EmailValidatorServiceImpl {
+package mate.academy.service.impl;
+
+import java.util.regex.Pattern;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Service;
+import mate.academy.service.EmailValidatorService;
+
+@Service
+public class EmailValidatorServiceImpl implements EmailValidatorService {
+    private static final Pattern EMAIL_PATTERN = Pattern.compile("\\w+@\\w+\\.\\w+");
+
+    @Override
+    public boolean isValid(String email) throws RegistrationException {

```

Comment:
It is not correct to throw RegistrationException when the email is not valid. You can as well create some custom data validation exception for this case;